### DISCUSSION TITLE
How to chnge direction of flow from fan boundary

### DISCUSSION THREAD

**Discussion by gtrewq** [nextfoam/baram#109] [Q&A]
Hello,

I'm trying to run simulation case with fan boundary condition. The simulation was completed without error, but the direction of the airflow from the fan boundary was opposite to the intended direction. I will appreciate you if you give the information for changing direction of flow from fan boundary.

Best regards,
Junichi Fukui

**Comment by kdg5424** [comment]
Hello Junichi. Sorry for the late reply.
In current situation, we haven't implemented the function which can change flow direction of the fan in GUI.
So I'll let you know how to configure to change the flow direction of the fan as soon as possible in TUI.
I'm very sorry for the inconvenience.

Regards

Donggyu kim

**Reply by gtrewq** [reply]
Hello Kim,

Thank you for your reply.

Changing the settings in the TUI is not a problem for me.  I would be grateful if you could show me how to do this.

Best regards,
Junichi Fukui

**Reply by kdg5424** [reply]
Hell Junichi. We're testing how to change it in TUI.
Sorry for the delay. This week is the holiday season in Korea. So the delay happens.
I'll let you know as soon as possible.

Regards
Donggyu kim

**Reply by gtrewq** [reply]
Hello Kim,

Thank you for replying to this matter despite the holiday season. No problem. It would be great if you could tell me how to set this up in the next week.

Have a nice holiday!

Best regards,
Junichi Fukui

**Comment by kdg5424** [comment]
Hello Fukui. I'm really sorry for the tremendous delay.

There is a critical bug with code which flow direction of fan is decided from **master face** to **slave face**.
We'll fix it as quickly but it will take a lot of time. Even I'm not sure how it will be in current status.
I'm really sorry for the inconvenience.

Best regards

Donggyu kim
